frontend-ssr/components/exercise-submissions/ExerciseSubmissionSkeleton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './ExerciseSubmission.module.scss';
+
+export default function SubmissionSkeleton() {
+  return (
+    <div
+      className={`${styles.card} ${styles['skeleton-screens']} text-white rounded-md bg-grey`}
+    >
+      <div className={`${styles['card-top']} d-flex align-items-center justify-content-between`}>
+        <div className={`${styles['card-info']} d-flex align-items-center justify-content-between`}>
+          <a
```

Comment:
This one is already an <a>, right? I don't really understand the problem here.

Reply:
Oups, my bad. I meant replace this `<a>`. No point in a having a link pointing to nowhere....remember we don't know the content beforehand.